src/main/webapp/WEB-INF/view/navbar.jsp

```diff
@@ -9,7 +11,7 @@
     <a href="/about.jsp">About</a>
     <a href="/activity">Activity</a>
     <% String username = (String) request.getSession().getAttribute("user");
-    if(username !=null && username.equals("adminama")){
+    if(username !=null && UserStore.getInstance().hasAdmin(username)){
```

Comment:
I'm not sure if it is useful in all the other callers of hasAdmin(), but it might be worth looking into allowing this method to take null and just return false. It would simplify the logic here. No change needed for this pull request, but perhaps something to consider for a followup one.